engine/runtime/src/main/java/org/enso/interpreter/runtime/Context.java

```diff
@@ -258,6 +258,18 @@ private void renamePackages(String oldName, String newName) {
     return getTopScope().getModule(moduleName);
   }
 
+  /**
+   * Fetches a module with a given name.
+   *
+   * @param moduleName the qualified name of the module to lookup.
+   * @return the relevant module.
+   */
+  public Module getModule(String moduleName) throws ModuleDoesNotExistException {
```

Comment:
Please move this to some support class that's associated with the execution server directly.
This API is already quite large, so one-user-module-specific helpers should be extracted.